en/docs/learn/extension-points-for-oauth.md

```diff
@@ -62,7 +58,9 @@ Restart the server to effect changes.
 <tr class="odd">
 <td>Usage</td>
 <td><p>This extension point can be used to implement a grant validator that validates the grant request sent to the ' <em>/oauth2/token</em> ' endpoint. Request parameters and headers can be validated by implementing this extension point.<br />
-If a new grant type is being implemented and defined with a new grant type identifier with the <a href="#oauth-grant-handler">OAuth Grant Handler</a> extension, you must also implement a grant validator and register that against the same grant type identifier from the <em>identity.xml</em> .<br />
+If a new grant type is being implemented and defined with a new grant type identifier with the <a 
+href="#oauth-grant-handler">OAuth Grant Handler</a> extension, you must also implement a grant validator and register
```

Comment:
shall we change the href here to the full path as: ../../subfolder/filename#oauth-grant-handler